src/table/table.component.test.tsx

```diff
@@ -203,14 +208,31 @@ describe('Table', () => {
     expect(onSort).toHaveBeenCalledWith('shotNum', null);
   });
 
+  it("updates columns when a column's word wrap is toggled", () => {
+    createView();
+
+    let menuIcon = screen.getByLabelText('timestamp menu');
+    fireEvent.click(menuIcon);
+
+    expect(screen.getByText('Turn word wrap on')).toBeInTheDocument();
```

Comment:
Not an issue but I think the docs say that, if you're specifically checking for an element to be present on-screen, you can just write this as
```
screen.getByText('my text');
```
Although if you're expecting the opposite, you need to be explicit so you actually need
```
expect(screen.queryByText('my text').not.toBeInTheDocument();
```

Reply:
I don't know if it's a style thing but I think I prefer the explicit assertation - either way it's good to know you can do it either way